packages/xod-client/src/debugger/containers/Breadcrumbs.jsx

```diff
@@ -8,31 +8,38 @@ import { bindActionCreators } from 'redux';
 import { drillDown } from '../actions';
 import { getRenerableBreadcrumbChunks, getBreadcrumbActiveIndex } from '../../editor/selectors';
 
-const Breadcrumbs = ({ chunks, activeIndex, actions }) => (
-  <ul className="Breadcrumbs Breadcrumbs--debugger">
-    {chunks.map((chunk, i) => {
-      const cls = classNames('Breadcrumbs-chunk-button', {
-        'is-active': (i === activeIndex),
-        'is-tail': (i > activeIndex),
-      });
-      return (
-        <li key={chunk.nodeId}>
-          <button
-            className={cls}
-            onClick={() => actions.drillDown(chunk.patchPath, chunk.nodeId)}
-          >
-            {chunk.label}
-          </button>
-        </li>
-      );
-    })}
-  </ul>
+const Breadcrumbs = ({ chunks, activeIndex, actions, children }) => (
+  <div className="Breadcrumbs Breadcrumbs--debugger">
+    <ul>
+      {chunks.map((chunk, i) => {
+        const cls = classNames('Breadcrumbs-chunk-button', {
+          'is-active': (i === activeIndex),
+          'is-tail': (i > activeIndex),
+        });
+        return (
+          <li key={chunk.nodeId}>
+            <button
+              className={cls}
+              onClick={() => actions.drillDown(chunk.patchPath, chunk.nodeId)}
+            >
+              {chunk.label}
+            </button>
+          </li>
+        );
+      })}
+    </ul>
+    {children}
+  </div>
 );
 
 Breadcrumbs.propTypes = {
   chunks: PropTypes.arrayOf(PropTypes.object),
   activeIndex: PropTypes.number,
   actions: PropTypes.objectOf(PropTypes.func),
+  children: PropTypes.oneOfType([
+    PropTypes.arrayOf(PropTypes.node),
+    PropTypes.node,
```

Comment:
`PropTyes.node` is "Anything that can be rendered: numbers, strings, elements or an array (or fragment) containing these types.". No need for `arrayOf`.